src/controls/NumericControl.tsx

```diff
@@ -0,0 +1,110 @@
+import { ControlProps } from "@jsonforms/core"
+import { Col, Form, InputNumber, Row, Slider } from "antd"
+import { decimalToPercentage, percentageStringToDecimal } from "./utils"
+
+
+
+type NumericControlProps = Omit<ControlProps, "data"> & {
```

Comment:
I had to add this `Omit` to make eslint happy because it seems to have more rules turned on than in mercury-web